Content.Server/GameTicking/Rules/Components/ZombieRuleComponent.cs

```diff
@@ -25,4 +25,21 @@ public sealed partial class ZombieRuleComponent : Component
     /// </summary>
     [DataField]
     public float ZombieShuttleCallPercentage = 0.7f;
+
+    /// <summary>
+    /// Is the automatic shuttle called.
+    /// </summary>
+    public bool? ZombieShuttleCalled;
```

Comment:
It feels weird to have a nullable boolean. 

You only ever set it to true (when the shuttle is called), so there's only really two states you're using: true (called) or null (not called). Instead of using null, you should just use false.

```suggestion
    public bool ZombieShuttleCalled;
```

Reply:
i made it nullable just to use in the next code
![image](https://github.com/user-attachments/assets/df91b594-4edf-4737-809f-4804ed1b70ac)

i suppose with it help, it chooses only components that has the indicated bool
and i'm not sure if it's gonna work if i set it to false and make check for false

Reply:
Is ZombieShuttleCalled ever false though? 

If it’s not null, it is always true (shuttle is called), otherwise it’s null (shuttle not called)

You’ve basically just turned nullability into a Boolean, when you could’ve just used a regular Boolean.